include/aws/checksums/private/crc_util.h

```diff
@@ -62,4 +62,18 @@ static inline uint64_t aws_bswap64_if_be(uint64_t x) {
 #endif
 }
 
+/**
+ * Force resolution of any global variables for CRC32.
+ * Note: in usual flow those are resolved on the first call to crc32 functions, which
+ * which might be deemed non-thread safe by some tools.
+ */
+AWS_CHECKSUMS_API void aws_checksums_crc32_init(void);
```

Comment:
these functions are purely internal

```suggestion
void aws_checksums_crc32_init(void);
```